delfin/db/sqlalchemy/models.py

```diff
@@ -267,17 +267,22 @@ class Task(BASE, DelfinBase):
     args = Column(JsonEncodedDict)
     last_run_time = Column(Integer)
     job_id = Column(String(36))
```

Comment:
Hi, I am confused about the job_id in the Task, where to find the job schema?

Reply:
job_id is the id in scheduler for the job posted for this task entry. This id can be used to manage the job, like removal of job